wagtail/contrib/modeladmin/templates/modeladmin/index.html

```diff
@@ -17,45 +17,30 @@
-                                {{ view.get_page_title }}
-                                {% if view.get_page_subtitle %} <span class="w-header__subtitle">{{ view.get_page_subtitle }}</span> {% endif %}
-                                {% include 'modeladmin/includes/result_count.html' %}
-                            </h1>
-                        {% endblock %}
-                    </div>
-                    {% block search %}{% search_form %}{% endblock %}
-                </div>
-                {% block header_extra %}
-                    <div class="right header-right">
-                        {% if view.list_export or user_can_create %}
-                            <div class="actionbutton">
-                                {% if user_can_create %}
-                                    {% include 'modeladmin/includes/button.html' with button=view.button_helper.add_button %}
-                                {% endif %}
-                                {% if view.list_export %}
-                                    <div class="dropdown dropdown-button match-width col">
-                                        <a href="?export=xlsx&{{ request.GET.urlencode }}" class="button bicolor button--icon">{% icon name="download" wrapped=1 %}{% trans 'Download XLSX' %}</a>
-                                        <div class="dropdown-toggle">{% icon name="arrow-down" %}</div>
-                                        <ul>
-                                            <li><a  class="button bicolor button--icon" href="?export=csv&{{ request.GET.urlencode }}">{% icon name="download" wrapped=1 %}{% trans 'Download CSV' %}</a></li>
-                                        </ul>
-                                    </div>
-                                {% endif %}
+        {% fragment as description %}
+            {% include 'modeladmin/includes/result_count.html' %}
+        {% endfragment %}
+        {% fragment as header_extra %}
+            <div class="right header-right">
+                {% if view.list_export or user_can_create %}
+                    <div class="actionbutton">
+                        {% if user_can_create %}
+                            {% include 'modeladmin/includes/button.html' with button=view.button_helper.add_button %}
+                        {% endif %}
+                        {% if view.list_export %}
+                            <div class="dropdown dropdown-button match-width col">
+                                <a href="?export=xlsx&{{ request.GET.urlencode }}" class="button bicolor button--icon">{% icon name="download" wrapped=1 %}{% trans 'Download XLSX' %}</a>
+                                <div class="dropdown-toggle">{% icon name="arrow-down" %}</div>
+                                <ul>
+                                    <li><a  class="button bicolor button--icon" href="?export=csv&{{ request.GET.urlencode }}">{% icon name="download" wrapped=1 %}{% trans 'Download CSV' %}</a></li>
```

Comment:
While we are adjusting, can we fix up this extra white-space here.

```suggestion
                                    <li><a class="button bicolor button--icon" href="?export=csv&{{ request.GET.urlencode }}">{% icon name="download" wrapped=1 %}{% trans 'Download CSV' %}</a></li>
```